### DISCUSSION TITLE
There is no animation effect of customtkinter at all, and there are no more people who develop some numberanimate animations like javascript

### DISCUSSION THREAD

**Discussion by maxwangwyf** [TomSchimansky/CustomTkinter#2568] [General]
There is no animation effect of customtkinter at all, and there are no more people who develop some numberanimate animations like javascript
![numberAnimate](https://github.com/user-attachments/assets/a686fee0-c9fc-4d68-a282-29dab5ee35b2)

**Comment by maxwangwyf** [comment]
please

**Reply by maxwangwyf** [reply]
use ctk.canvas to make

**Reply by Akkiraj1234** [reply]
u can use ctk.canvas or after and place methods for moving for simple animation etc.. can be make like that
